CMakeLists.txt

```diff
@@ -122,30 +122,16 @@ endif ()
 
 
 #--------------------------------------------------------------------------------------------------
-# Extract version string and version components from Git repository.
+# Auto-generate version header.
 #--------------------------------------------------------------------------------------------------
 
-# Extract version string from Git repository.
-execute_process (
-    COMMAND ${git_command} describe --long
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    RESULT_VARIABLE git_describe_result
-    OUTPUT_VARIABLE appleseed_version_string
-    ERROR_QUIET
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# Use dummy version string if 'git describe' command failed (e.g. when building from a source dump).
-if (NOT ${git_describe_result} EQUAL 0)
-    set (appleseed_version_string "0.0.0")
-endif ()
-
-# Extract version components from version string.
-string (REGEX MATCHALL "[0-9]+" appleseed_version_components ${appleseed_version_string})
-list (GET appleseed_version_components 0 appleseed_version_major)
-list (GET appleseed_version_components 1 appleseed_version_minor)
-list (GET appleseed_version_components 2 appleseed_version_patch)
+set (appleseed_version_major 1)
+set (appleseed_version_minor 1)
+set (appleseed_version_patch 0)
+set (appleseed_version_alpha 24)
```

Comment:
My motivation for adding this is to be able to check the version numbers in code that uses the apleseed library and decide from that which features are available. Keeping client code backwards compatible but being able to use new features when they are added.
Having five version numbers is just too much. We need to simplify a bit.